src/annotator/annotation-counts.ts

```diff
@@ -17,7 +18,7 @@ const ANNOTATION_COUNT_ATTR = 'data-hypothesis-annotation-count';
  */
 export function annotationCounts(
   rootEl: Element,
-  rpc: PortRPC<'publicAnnotationCountChanged', string>,
+  rpc: PortRPC<SidebarToHostCalls, CallMap>,
```

Comment:
Using the `CallMap` type here means "we're not going to make any calls on this port".